packages/wonder-blocks-pill/src/index.ts

```diff
@@ -1,2 +1,3 @@
-// Just a temporary export to have some code.
-export const NOTHING = "NOTHING";
+import Pill from "./components/pill";
+
+export {Pill as default};
```

Comment:
Great question! I think they both work, and I see both in the codebase. I'll change it to `export default Pill` though since that's more common to see.